externs/olx.js

```diff
@@ -2672,6 +2672,42 @@ olx.interaction.SelectOptions.prototype.multi;
 
 
 /**
+ * Options for snap
+ * @typedef {{
+ *     features: (Array.<ol.Feature>|ol.Collection.<ol.Feature>|undefined),
```

Comment:
I would not allow an `Array.<ol.Feature>`. `ol.interaction.Draw` also does not, and an array does not allow applications to listen to changes.